litmus-portal/frontend/src/components/Header/styles.ts

```diff
@@ -64,7 +65,7 @@ const useStyles = makeStyles((theme: Theme) => ({
   // Style for ProfileDropdownSection and ProfileDropdownItems.
   avatarBackground: {
     color: theme.palette.customColors.white(1),
-    backgroundColor: theme.palette.secondary.main,
+    backgroundColor: theme.palette.totalRunsCountColor,
```

Comment:
https://github.com/litmuschaos/litmus/pull/2120/files#r496476443

Reply:
As in headers we are using different bg colors for various sections, one value can't be inherited from the parent.